src/components/DeleteMovieDialog/DeleteMovieDialog.js

```diff
@@ -22,15 +22,24 @@ const DeleteMovieDialog = (props) => {
   };
 
   return (
-    <div className="delete-movie__dialog">
-      <div className="delete-movie__container">
-        <button className="close-button" onClick={closeDialog}>
-          <Icon path={mdiClose} size={"28px"} color="white" />
-        </button>
-        <h2>Delete movie</h2>
-        <p>Are you sure you want to delete this movie?</p>
-        <div className="delete-movie__container__button">
-          <button onClick={deleteMovieHandler}>Confirm</button>
+    <div className="movie-dialog">
+      <div className="modal-small">
```

Comment:
`movie` prefix -  it's better to have a generic modal as a basic component, and then build different types of components based on it, so I recommend a className change to `modal-dialog` - in react tasks we'll work on making a `ModalDialog` component, so let's make it easy for the future.

`modal-small` - it gives the wrong impression there's another modal somewhere that is bigger, so I recommend a change.

```suggestion
    <div className="modal-overlay"> // alternatively it can be a "modal" or "modal-container"
      <div className="modal-dialog"> 
```

PS. Whenever in doubt about names for base components, please refer to some common standards used in UI libraries such as [bootstrap](https://getbootstrap.com/docs/4.0/components/modal/) or [material-ui](https://mui.com/material-ui/react-modal/).

Reply:
All done.